### PR TITLE
Issue #233 add xunit test project

### PR DESCRIPTION
**Pull request by roarfred**
Did I mess up the solution file? See there's a lot of stuff under ConfigurationPlatforms...

### CODE REVIEW

**Comment by roarfred** [comment]
Appveyor didn't quite like this... There's some documentation on this here, saying there's [a way around by running as a test script](http://help.appveyor.com/discussions/problems/6501-systeminvalidoperationexception-unknown-test-framework-could-not-find-xunitdll-v1-or-xunitexecutiondll-v2)

From log:
> Discovering tests...OK
>%xunit20%\xunit.console.x86 "C:\projects\cbs\Source\VolunteerReporting\Tests\Web.Tests\bin\Release\netcoreapp2.0\Web.Tests.dll" -appveyor
> xUnit.net Console Runner (32-bit .NET 4.0.30319.42000)
> System.InvalidOperationException: Unknown test framework: could not find xunit.dll (v1) or xunit.execution.*.dll (v2) in C:\projects\cbs\Source\VolunteerReporting\Tests\Web.Tests\bin\Release\netcoreapp2.0
> Command exited with code 1

**Comment by roarfred** [comment]
Looks like this one was solved, thanks to @anderaus for asking the right question!
@karolikl, these are the steps and my findings:
* There are two appveyor.yml files, one under /Build/AppVeyor.yml and one under /Source/Project/AppVeyor.yml
* It turns out that testing was not disabled in the one under the project. Adding test:off at the end solved this
* Still, tests would not run. (Empty output under heading Run-Backend-Tests
* Added a line to log information, and it turned out it was using the default value
* Found that I could set the environment variable in my project's AppVeyor file, but it had no effect
* Now desperate, trying to make a dummy placeholder for the same variable name in the root AppVeyor, and VOILA!

**Comment by roarfred** [comment]
I'll make a separate PR for just these changes, so we can get it to the master without this one

**Comment by roarfred** [comment]
Please consider PR #260 first, this one builds upon the same changes

**Comment by roarfred** [comment]
Absolutely, I'll fix

File: Source/VolunteerReporting/Tests/Web.Tests/Web.Tests.csproj [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```
[the remark this answers is not in the document]
Comment (anderaus) [reply Source/VolunteerReporting/Tests/Web.Tests/Web.Tests.csproj]: I experienced the same issue. Had to target `netcoreapp2.0` in tests project to get `xunit.runner.visualstudio` to resolve. Haven't dug more into it - maybe create a separate issue for this (if it is an issue?).

[the remark this answers is not in the document]
Comment (einari) [reply Source/VolunteerReporting/Tests/Web.Tests/Web.Tests.csproj]: For test projects, netcoreapp2.0 is the way to go. netcoreapp2.0 includes netstandard2.0

**Comment by einari** [comment]
I suggest looking at my comment in #255 for a common props file for test projects.

**Pull request merged**